src/category_theory/functor/basic.lean

```diff
@@ -82,13 +84,12 @@ variables {C : Type u₁} [category.{v₁} C]
 /--
 `F ⋙ G` is the composition of a functor `F` and a functor `G` (`F` first, then `G`).
 -/
-def comp (F : C ⥤ D) (G : D ⥤ E) : C ⥤ E :=
+@[simps obj] def comp (F : C ⥤ D) (G : D ⥤ E) : C ⥤ E :=
 { obj := λ X, G.obj (F.obj X),
   map := λ _ _ f, G.map (F.map f) }
 
 infixr ` ⋙ `:80 := comp
 
-@[simp] lemma comp_obj (F : C ⥤ D) (G : D ⥤ E) (X : C) : (F ⋙ G).obj X = G.obj (F.obj X) := rfl
 @[simp] lemma comp_map (F : C ⥤ D) (G : D ⥤ E) {X Y : C} (f : X ⟶ Y) :
```

Comment:
Can this one be given by `simps` too?

Reply:
No; we want `X Y` implicit, and simps won't do this.